pygments/lexers/scdoc.py

```diff
@@ -0,0 +1,56 @@
+import re
+
+from pygments.lexer import RegexLexer, include, bygroups, \
+    using, this
+from pygments.token import Text, Comment, Keyword, String, \
+    Generic
+
+
+__all__ = ['ScdocLexer']
+
+
+class ScdocLexer(RegexLexer):
+    name = 'scdoc'
+    aliases = ['scdoc', 'scd']
+    filenames = ['*.scd', '*.scdoc']
+    flags = re.MULTILINE
+
+    tokens = {
+        'root': [
+            # comment
+            (r'^(; .+\n)', bygroups(Comment)),
```

Comment:
So a line with only `;` is not a valid comment?

Reply:
yep.

https://git.sr.ht/~sircmpwn/scdoc/tree/master/scdoc.5.scd#L194

> Lines beginning with ; and a space are ignored.

Reply:
But I relax it and remove the space I guess.